File: fizzbuzz.cpp
#include <iostream>
using namespace std;


int i;
void f(int a,int b)
{
    
    for(i=a;i<=b;i++)
    {
        if(i%3==0 &&  i%5==0)
            cout<<"FizzBuzz"<<endl;
        else if(i%3==0)
            cout<<"Buzz"<<endl;
        else if(i%5 == 0)
            cout<<"Fizz"<<endl;
        else cout<<i<<endl;
    }
    
    
}
int main()
{
    int k,j;
    cin>>k;
    cin>>j;
    f(k,j);
    
}

File: fizzbuzz.js
console.log("This a javascript automatically generate an array from number 10 to 250, then For each number in the sequence, output the number to the screen.However, if the number is divisible by 3, output the word Buzz instead of the number. If the number is divisble by 5, output Fizz to the screen instead of the number. If the number is divisible by both, output BuzzFizz to the screen instead of the number. ");;

for (i=10; i<=250; i++) {
	if (i % 3 == 0) {
		if (i % 5 ==0) {
			console.log('FizzBuzz');
		} else {
			console.log('Fizz');
		}
	} else if(i % 5 == 0) {
			console.log('Buzz');
	} else
			console.log(i);
}
